api/Document.json

```
@@ -8901,6 +8901,53 @@
             "standard_track": true,
             "deprecated": true
           }
+        },
+        "accepts_TrustedHTML": {
+          "__compat": {
+            "description": "Accepts `TrustedHTML` instances",
+            "mdn_url": "https://developer.mozilla.org/docs/Web/API/Document/write",
+            "spec_url": "https://html.spec.whatwg.org/multipage/dynamic-markup-insertion.html#dom-document-write",
```

Let's link closer to where TrustedHTML is actually mentioned:

```suggestion
            "spec_url": [
              "https://html.spec.whatwg.org/multipage/dynamic-markup-insertion.html#document-write-steps",
              "https://html.spec.whatwg.org/multipage/dynamic-markup-insertion.html#:~:text=if%20value%20is%20a%20trustedhtml%20object"
            ],
```